rocketpy/simulation/flight.py

```diff
@@ -417,6 +417,8 @@ class Flight:
         of frequency in Hz. Can be called or accessed as array.
     Flight.static_margin : Function
         Rocket's static margin during flight in calibers.
+    Flight.stability_margin : rocketpy.Function
```

Comment:
Just so that this follows the patterns of all other attributes. In the future, we should probably rename all of them.

```suggestion
    Flight.stability_margin : Function
```

Reply:
you're right. Both of them works very well in the docs, properly creating the reference to the Function class.